{"failedTests":[{"testName":"parallel/test-http2-socket-close","timestamp":1725936134163,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win10-COMPILED_BY-vs2022 #30148","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/30148/","buildNumber":30148,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"07421203e08a47906069c80100cb86a94e74c4a0","callStack":[{"upstreamBuild":65751,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74240,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62218,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54841"}],"tap":{"ok":false,"name":"parallel/test-http2-socket-close","id":492,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"node:events:498\n      throw er; // Unhandled 'error' event\n      ^\n\nError: read ECONNRESET\n    at TLSWrap.onStreamRead (node:internal/stream_base_commons:216:20)\nEmitted 'error' event on ClientHttp2Stream instance at:\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {\n  errno: -4077,\n  code: 'ECONNRESET',\n  syscall: 'read'\n}\n\nNode.js v23.0.0-pre"},"time":1861.027,"fullname":"parallel/test-http2-socket-close","closingTestPoint":false}},{"testName":"parallel/test-runner-run-watch","timestamp":1725936134163,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win10-COMPILED_BY-vs2022 #30148","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/30148/","buildNumber":30148,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"07421203e08a47906069c80100cb86a94e74c4a0","callStack":[{"upstreamBuild":65751,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74240,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62218,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54841"}],"tap":{"ok":false,"name":"parallel/test-runner-run-watch","id":903,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"timeout\n▶ test runner watch mode\n  ✖ should run tests repeatedly (9097.664ms)\n    AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n    \n    '# Subtest: test.js\\n' +\n      'not ok 2 - test.js\\n' +\n      '  ---\\n' +\n      '  duration_ms: 1135.8036\\n' +\n      \"  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n\" +\n      \"  failureType: 'testCodeFailure'\\n\" +\n      '  exitCode: ~\\n' +\n      \"  signal: 'SIGTERM'\\n\" +\n      \"  error: 'test failed'\\n\" +\n      \"  code: 'ERR_TEST_FAILURE'\\n\" +\n      '  ...\\n' +\n      '# Subtest: test.js\\n' +\n      'not ok 3 - test.js\\n' +\n      '  ---\\n' +\n      '  duration_ms: 943.1098\\n' +\n      \"  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n\" +\n      \"  failureType: 'testCodeFailure'\\n\" +\n      '  exitCode: ~\\n' +\n      \"  signal: 'SIGTERM'\\n\" +\n      \"  error: 'test failed'\\n\" +\n      \"  code: 'ERR_TEST_FAILURE'\\n\" +\n      '  ...\\n' +\n      '# Subtest: test has ran\\n' +\n      'ok 3 - test has ran\\n' +\n      '  ---\\n' +\n      '  duration_ms: 10.5893\\n' +\n      '  ...\\n' +\n      '# Subtest: test.js\\n' +\n      'not ok 4 - test.js\\n' +\n      '  ---\\n' +\n      '  duration_ms: 1040.3016\\n' +\n      \"  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n\" +\n      \"  failureType: 'testCodeFailure'\\n\" +\n      '  exitCode: ~\\n' +\n      \"  signal: 'SIGTERM'\\n\" +\n      \"  error: 'test failed'\\n\" +\n      \"  code: 'ERR_TEST_FAILURE'\\n\" +\n      '  ...\\n' +\n      '# Subtest: test.js\\n' +\n      'not ok 5 - test.js\\n' +\n      '  ---\\n' +\n      '  duration_ms: 984.7539\\n' +\n      \"  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n\" +\n      \"  failureType: 'testCodeFailure'\\n\" +\n      '  exitCode: ~\\n' +\n      \"  signal: 'SIGTERM'\\n\" +\n      \"  error: 'test failed'\\n\" +\n      \"  code: 'ERR_TEST_FAILURE'\\n\" +\n      '  ...\\n' +\n      '# Subtest: test.js\\n' +\n      'not ok 6 - test.js\\n' +\n      '  ---\\n' +\n      '  duration_ms: 995.606\\n' +\n      \"  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n\" +\n      \"  failureType: 'testCodeFailure'\\n\" +\n      '  exitCode: ~\\n' +\n      \"  signal: 'SIGTERM'\\n\" +\n      \"  error: 'test failed'\\n\" +\n      \"  code: 'ERR_TEST_FAILURE'\\n\" +\n      '  ...\\n' +\n      '# Subtest: test has ran\\n' +\n      'ok 7 - test has ran\\n' +\n      '  ---\\n' +\n      '  duration_ms: 10.8934\\n' +\n      '  ...\\n' +\n      '1..7\\n' +\n      '# tests 7\\n' +\n      '# suites 0\\n' +\n      '# pass 2\\n' +\n      '# fail 5\\n' +\n      '# cancelled 0\\n' +\n      '# skipped 0\\n' +\n      '# todo 0\\n' +\n      '# duration_ms 972.9421\\n'\n    \n        at testUpdate (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:81:14)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async testWatch (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:173:26)\n        at async TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:183:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 0)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: true,\n      code: 'ERR_ASSERTION',\n      actual: \"# Subtest: test.js\\nnot ok 2 - test.js\\n  ---\\n  duration_ms: 1135.8036\\n  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n  failureType: 'testCodeFailure'\\n  exitCode: ~\\n  signal: 'SIGTERM'\\n  error: 'test failed'\\n  code: 'ERR_TEST_FAILURE'\\n...\",\n      expected: /tests 1/,\n      operator: 'match'\n    }"},"time":122109.758,"fullname":"parallel/test-runner-run-watch","closingTestPoint":false}}]}